src/Components/Books.css

```diff
@@ -11,6 +11,12 @@
   margin: 0;
 }
 
+h3 {
+  margin: 0;
+  padding: 0;
+  font-size: 10px;
+}
+
```

Comment:
Hi @Divinity-dev 

Nice job implementing the different aspects of the project requirements but you are missing the design match

- [ ] Please your styling should be exactly the same as the one provided on [Zeplin](https://zeplin.io/) design.
![zeplin](https://user-images.githubusercontent.com/55596970/224457436-ccbbfb93-9b82-4f72-b597-eecaba4e9626.png)
**Your Design**
![yours](https://user-images.githubusercontent.com/55596970/224457631-9f10aa55-3f74-4c27-92aa-714d8bdd940d.png)